opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -416,7 +424,10 @@ def _initialize_components(
     id_generator: IdGenerator | None = None,
     setup_logging_handler: bool | None = None,
     exporter_args_map: ExporterArgsMap | None = None,
+    span_processor: Type[SpanProcessor] | None = None,
```

Comment:
This is the same for every exporter, is it enough or we want to make it per exporter?